src/routing/Router.tsx

```diff
@@ -30,6 +25,19 @@ export default function FormRouter(props: RouterProps): JSX.Element {
         <Routes>{props.children}</Routes>
       </Formik>
       <FormFooter />
+    </>
+  );
+}
+
+/**
+ * Manages form pages as routes
+ * Parent formik insance is rendered here
+ * @beta
+ */
+export default function FormRouter(props: RouterProps): JSX.Element {
+  return (
+    <BrowserRouter basename={props.basename}>
+      <FormRouterInternal {...props} />
```

Comment:
Was this InternalRouter only added for testing purposes? Also is the FormRouterInternal suppose to be used by users?

Reply:
@bjmccotter7192 I went ahead and moved this into the tests since it is only being used there